packages/ui/src/components/Table/TablePagination.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, TableCell, TableFooter, TableRow } from '@mui/material';
+import usePagination from '@mui/material/usePagination/usePagination';
+import { FC } from 'react';
+
+import { TablePaginationItem } from './TablePaginationItem';
+import { PageChangeHandler } from './types';
+
+interface TablePaginationProps {
+    itemsCount: number;
+    currentPage: number;
+    rowsPerPage: number;
+    onChangePage?: PageChangeHandler;
+}
+
+export const TablePagination: FC<TablePaginationProps> = ({ itemsCount, currentPage, rowsPerPage, onChangePage }) => {
```

Comment:
I would also add a storybook story for this component alone to showcase it

Reply:
Why so? I would use the component only with the BaseTable. And we have one for it.

Reply:
You are right, missed the fact that we use this as the default now 👍